docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apk add --update --no-cache git openssh make gcc g++ python curl libpng-dev
 # Clone repo
 RUN mkdir -p /app/warframe-items \
   && cd app \
-  && git clone https://github.com/WFCD/warframe-items \
```

Comment:
i am worried as this will only ever build development, not master... i guess if the whole process was changed to exclude master, this wouldn't be a problem